listenbrainz/webserver/login/provider.py

```diff
@@ -32,13 +33,27 @@ def get_user():
     data = s.get('oauth2/userinfo').json()
     musicbrainz_id = data.get('sub')
     musicbrainz_row_id = data.get('metabrainz_user_id')
-    user = db_user.get_or_create(musicbrainz_row_id, musicbrainz_id)
-    if user:
-        if not user['musicbrainz_row_id']:
-            db_user.update_musicbrainz_row_id(musicbrainz_id, data['metabrainz_user_id'])
-        return User.from_dbrow(user)
+
+    if not current_app.config["DEBUG"]:
```

Comment:
I don't think that we should use this as the guard for identifying if we connect to the musicbrainz db or not. Let's instead use a connection/engine object, which gets set if the connection string is available in the config.

It'd be great to be able to connect to a real mb database to be able to test this during development, too. Can we have a system using a docker-compose overlay file that adds a new session? I guess the test database won't be good for this because it won't have user emails, right?

Reply:
fixed now.